packages/jest-config/src/readConfigFileAndSetRootDir.ts

```diff
@@ -89,13 +89,19 @@ export default async function readConfigFileAndSetRootDir(
 }
 
 // Load the TypeScript configuration
+let extraTSLoaderOptions: Record<string, unknown>;
+
 const loadTSConfigFile = async (
   configPath: string,
 ): Promise<Config.InitialOptions> => {
   // Get registered TypeScript compiler instance
   const docblockPragmas = parse(extract(fs.readFileSync(configPath, 'utf8')));
   const tsLoader = docblockPragmas['jest-config-loader'] || 'ts-node';
```

Comment:
should we also change this default behavior of using ts-node if no loader is passed in a separate PR?

Reply:
that's a separate thing, yeah. happy to take a PR